plasmapy/physics/distribution.py

```diff
@@ -143,6 +143,144 @@ def Maxwellian_1D(v,
         return distFunc
 
 
+def Maxwellian_velocity_2D(vx,
+                           vy,
+                           T,
+                           particle="e",
+                           Vx_drift=0,
+                           Vy_drift=0,
+                           vTh=np.nan,
+                           units="units"):
+    r"""
+    pdf of velocity for a 2D Maxwellian distribution.
+
+    Return the probability of finding a particle with velocity components
```

Comment:
Technically this returns the probability density function, not the probability (to get a probability you have to multiply by `dv_x . dv_y`).

Reply:
done